buildsupport/httpclient/pom.xml

```diff
@@ -33,7 +33,7 @@
       <dependency>
         <groupId>org.apache.httpcomponents</groupId>
         <artifactId>httpclient</artifactId>
-        <version>4.3.5</version>
+        <version>4.3.6</version>
```

Comment:
why?  this is used in 2 places? If it grows to more perhaps, but we don't need properties to manage versions for all dependencies, especially not in these tiny buildsupport modules.